sql/update/add-manager-user-group.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO user_group VALUES (3,'manager');
```

Comment:
Please, add a small explanation comment on this file, like in other update files.